pkg/auth/tokens/token_util.go

```diff
@@ -59,6 +60,16 @@ func IsExpired(token v3.Token) bool {
 	return durationElapsed.Seconds() >= ttlDuration.Seconds()
 }
 
+// IsIdleExpired checks if the idle session timeout was reached since last update.
+func IsIdleExpired(token v3.Token, lastTimeActivity metav1.Time) bool {
+	// useractivity has not been initialized yet
```

Comment:
nit: Feel free to ignore , but I'd probably remove the comment as the code is self-evident.
```suggestion
```